### ISSUE TITLE
Problem while extracting the subjects from Graph

### ISSUE THREAD

**Issue by shahbazwasti** [spyder-ide/spyder#6825]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Users\Shahbaz\AppData\Local\Continuum\anaconda3\lib\site-packages\qtconsole\base_frontend_mixin.py", line 163, in _dispatch
    handler(msg)
  File "C:\Users\Shahbaz\AppData\Local\Continuum\anaconda3\lib\site-packages\spyder\widgets\ipythonconsole\namespacebrowser.py", line 220, in _handle_execute_reply
    self.handle_exec_method(msg)
  File "C:\Users\Shahbaz\AppData\Local\Continuum\anaconda3\lib\site-packages\spyder\widgets\ipythonconsole\shell.py", line 354, in handle_exec_method
    view = ast.literal_eval(literal)
  File "C:\Users\Shahbaz\AppData\Local\Continuum\anaconda3\lib\ast.py", line 85, in literal_eval
    return _convert(node_or_string)
  File "C:\Users\Shahbaz\AppData\Local\Continuum\anaconda3\lib\ast.py", line 66, in _convert
    in zip(node.keys, node.values))
  File "C:\Users\Shahbaz\AppData\Local\Continuum\anaconda3\lib\ast.py", line 65, in <genexpr>
    return dict((_convert(k), _convert(v)) for k, v
  File "C:\Users\Shahbaz\AppData\Local\Continuum\anaconda3\lib\ast.py", line 66, in _convert
    in zip(node.keys, node.values))
  File "C:\Users\Shahbaz\AppData\Local\Continuum\anaconda3\lib\ast.py", line 65, in <genexpr>
    return dict((_convert(k), _convert(v)) for k, v
  File "C:\Users\Shahbaz\AppData\Local\Continuum\anaconda3\lib\ast.py", line 84, in _convert
    raise ValueError('malformed node or string: ' + repr(node))
ValueError: malformed node or string: <_ast.Call object at 0x000001E7CE236320>

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.2.8). Please update by running in the Anaconda prompt

    conda update spyder

**Comment by CAM-Gerlach** [comment]
Duplicate of #5636 ; fixed in Spyder 3.2.6.
